packages/generators/__tests__/init-generator.test.ts

```diff
@@ -1,7 +1,7 @@
 import { join } from 'path';
```

Comment:
You should install also `@types/yeoman-assert`

Reply:
Right, will make the changes, good catch. 👍